precompiles/prototype/IPrototype.go

```diff
@@ -135,11 +134,11 @@ func NewIPrototypeFilterer(address common.Address, filterer bind.ContractFiltere
 
 // bindIPrototype binds a generic wrapper to an already deployed contract.
 func bindIPrototype(address common.Address, caller bind.ContractCaller, transactor bind.ContractTransactor, filterer bind.ContractFilterer) (*bind.BoundContract, error) {
```

Comment:
yes , I think this might be caused by by a different version of solc on my local machine , Ill probably need to verify and fix this 
I am using` 0.8.26 `